2022-12-05 21:42:29,102 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 137, in onUpdateGUI
    print(self.getInputList())
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 158, in getInputList
    input_str_list.append('ANA' + str(index), +'_' + str(index+1)+'_FullyDifferentiall')
TypeError: bad operand type for unary +: 'str'

2022-12-05 21:51:58,122 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 137, in onUpdateGUI
    print(self.getInputList())
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 169, in getInputList
    input_str_list.append('SingleEndANB{0}'.format(index*2)-8)
TypeError: unsupported operand type(s) for -: 'str' and 'int'

2022-12-05 21:51:58,125 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 137, in onUpdateGUI
    print(self.getInputList())
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 169, in getInputList
    input_str_list.append('SingleEndANB{0}'.format(index*2)-8)
TypeError: unsupported operand type(s) for -: 'str' and 'int'

2022-12-05 21:51:58,128 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 137, in onUpdateGUI
    print(self.getInputList())
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 169, in getInputList
    input_str_list.append('SingleEndANB{0}'.format(index*2)-8)
TypeError: unsupported operand type(s) for -: 'str' and 'int'

2022-12-05 21:55:18,614 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:55:18,616 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:55:18,618 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:55:18,620 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:55:18,624 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:55:18,627 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:55:18,629 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:55:18,631 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:56:05,457 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    if self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:56:05,459 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    if self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:56:05,460 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    if self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:56:05,461 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    if self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:56:05,463 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    if self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:56:05,465 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    if self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:56:05,466 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    if self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 21:56:05,468 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 138, in onUpdateGUI
    if self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:18,182 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:18,184 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:18,185 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:18,187 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:18,189 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:18,191 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:18,192 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:18,194 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:36,470 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 149, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:36,471 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 149, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:36,473 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 149, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:36,474 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 149, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:36,476 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 149, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:36,478 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 149, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:36,479 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 149, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:15:36,482 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 149, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:18:41,633 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:18:41,635 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:18:41,636 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:18:41,638 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:18:41,639 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:18:41,640 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:18:41,642 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:18:41,644 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 151, in onUpdateGUI
    for combobox in self.slot_comboboxes:
AttributeError: 'AdcTestWidget' object has no attribute 'slot_comboboxes'

2022-12-05 22:19:50,246 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 194, in onStart
    self.btnStart.setEanbled(False)
AttributeError: 'QPushButton' object has no attribute 'setEanbled'

2022-12-07 13:35:00,510 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 198, in onStart
    self.my_serial.adc_req(scan_mode, dma_en, scan_list)
TypeError: adc_req() missing 1 required positional argument: 'scan_list'

2022-12-07 13:35:48,917 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 198, in onStart
    self.my_serial.adc_req(scan_mode, dma_en, scan_list)
TypeError: adc_req() missing 1 required positional argument: 'scan_list'

2022-12-08 09:53:31,545 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 220, in onStart
    status = self.my_serial.adc_req(1, scan_mode, dma_en, scan_list)
  File "d:\56F81768_EVK\dsc_eval\my_serial.py", line 238, in adc_req
    self.sendMsg(0x0, PERF_MSG.ADC_REQ, msg_data, 0)
  File "d:\56F81768_EVK\dsc_eval\my_serial.py", line 83, in sendMsg
    self.__sendMsg(protocol, msg_id, msg, remote)
  File "d:\56F81768_EVK\dsc_eval\my_serial.py", line 110, in __sendMsg
    self.ser.write(frame)
  File "d:\56F81768_EVK\dsc_eval\eswenv\lib\site-packages\serial\serialwin32.py", line 317, in write
    raise SerialException("WriteFile failed ({!r})".format(ctypes.WinError()))
serial.serialutil.SerialException: WriteFile failed (PermissionError(13, 'The device does not recognize the command.', None, 22))

2022-12-08 11:11:54,265 {MainThread  ] {ERROR TypeError: onProcessStat() missing 1 required positional argument: 'node_info'

2022-12-08 11:12:48,423 {MainThread  ] {ERROR TypeError: onProcessStat() missing 1 required positional argument: 'node_info'

2022-12-08 11:12:48,427 {MainThread  ] {ERROR TypeError: onProcessStat() missing 1 required positional argument: 'node_info'

2022-12-08 11:13:32,034 {MainThread  ] {ERROR TypeError: onProcessStat() missing 1 required positional argument: 'node_info'

2022-12-08 11:14:15,201 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 253, in onProcessStat
    self.onAdcInd(data, node_info)
NameError: name 'node_info' is not defined

2022-12-08 15:05:24,830 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 219, in onStart
    scan_list[index] = combobox.itemData[index]
TypeError: 'builtin_function_or_method' object is not subscriptable

2022-12-08 15:05:49,665 {MainThread  ] {ERROR Traceback (most recent call last):
  File "d:\56F81768_EVK\dsc_eval\adc_test\adc_test.py", line 222, in onStart
    self.my_serial.adc_req(1, scan_mode, dma_en, scan_list)
  File "d:\56F81768_EVK\dsc_eval\my_serial.py", line 239, in adc_req
    msg_data = struct.pack('=BBB20B', start_stop, scan_modem, dma_en, *scan_list)
struct.error: required argument is not an integer

2023-02-14 14:10:06,946 {MainThread  ] {ERROR Traceback (most recent call last):
  File "/Users/jshi/Development/projects/SpeTester/dsc_eval/dsc_eval.py", line 120, in populateCombo
    for addr in self.serial_ports():
  File "/Users/jshi/Development/projects/SpeTester/dsc_eval/dsc_eval.py", line 104, in serial_ports
    s = serial.Serial(port)
AttributeError: module 'serial' has no attribute 'Serial'

2023-02-14 14:10:08,107 {MainThread  ] {ERROR Traceback (most recent call last):
  File "/Users/jshi/Development/projects/SpeTester/dsc_eval/dsc_eval.py", line 120, in populateCombo
    for addr in self.serial_ports():
  File "/Users/jshi/Development/projects/SpeTester/dsc_eval/dsc_eval.py", line 104, in serial_ports
    s = serial.Serial(port)
AttributeError: module 'serial' has no attribute 'Serial'

